data/kernel/post_process

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+ERROR=0
```

Comment:
to prevent unexpected problems I can recommend the line `set -euo pipefail` in all bash scripts by default.